src/backend/executor/execMain.c

```diff
@@ -266,6 +273,42 @@ CopyDirectDispatchFromPlanToSliceTable(PlannedStmt *stmt, EState *estate)
 	CopyDirectDispatchFromPlanToSliceTableWalker((Node *) stmt->planTree, &context);
 }
 
+static inline void
+UpdateCommandId(QueryDesc *queryDesc, int *prevCommandId, const char *functionName, bool trackStart)
+{
+	if (Gp_role != GP_ROLE_EXECUTE)
+	{
+		*prevCommandId = MyProc->queryCommandId;
+		MyProc->queryCommandId = queryDesc->command_id;
+	}
+
+#ifdef FAULT_INJECTOR
+	if (SIMPLE_FAULT_INJECTOR("track_query_command_id") == FaultInjectorTypeSkip ||
+		(trackStart && SIMPLE_FAULT_INJECTOR("track_query_command_id_at_start") == FaultInjectorTypeSkip))
+		elog(NOTICE, \
+			"START %s | Q: %s | QUERY ID: %d", \
+			functionName, \
+			queryDesc->sourceText, \
```

Comment:
Why is the backslash needed here? This is not a macro!

Reply:
At some early stage of development it was a macro. Agree, now it is redundant. Fixed it.